loadtest/control/actions.go

```diff
@@ -692,7 +688,7 @@ func Reload(u user.User) UserActionResponse {
 	}
 
 	// Getting unread teams.
-	_, err = u.GetTeamsUnread("")
+	_, err = u.GetTeamsUnread("", false)
```

Comment:
Wish we had the time to test CRT paths as well :disappointed:

Reply:
Yeah, along with other N recent features. Does reliable websockets ring a bell? :stuck_out_tongue_closed_eyes:  ([MM-38327](https://mattermost.atlassian.net/browse/MM-38327))